lua/rocks/operations.lua

```diff
@@ -125,6 +125,11 @@ operations.install = function(rock_spec, progress_handle)
                 runtime.packadd(name)
             end
 
+            -- Re-generate help tags
+            if config.generate_help_pages then
+                vim.cmd("helptags ALL")
+            end
+
```

Comment:
question: Wouldn't it make more sense to put this at the end of `operations.add` and `operations.sync`, instead of in `operations.install`?

That way, we only call it once, rather than multiple times for each plugin that is added, e.g. in `sync`.